Maquetado/scss/tech.scss

```diff
@@ -38,6 +38,19 @@
     z-index: -1;
   }
 
+  &:hover::before {
+    bottom: -10px;
+    border: 1px solid $green;
```

Comment:
you can remove this line, there's no need to set it again if it doesn't change from the original